tests/components/interactiveDataSources/LottieAnimation-test.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
Is there a way to refactor this test and use React Testing Library?  We're not writing new tests with Enzyme because React 18 doesn't support it.  We're eventually migrating our Enzyme tests to Testing Library.

Reply:
@anjenkin sure thing! @ValGreens561 and I just pushed some updates for these tests.